packages/batch-submitter/src/batch-submitter/tx-batch-submitter.ts

```diff
@@ -140,6 +140,10 @@ export class TransactionBatchSubmitter extends BatchSubmitter {
 
       if (!this.disableQueueBatchAppend) {
         const nonce = await this.signer.getTransactionCount()
+        // Generate the transaction we will repeatedly submit
+        const tx = await this.chainContract.populateTransaction.appendQueueBatch(
+          99999999
```

Comment:
This 9999999 constant really needs a comment or some kind of explanation, it's unclear why this number exists here in the code.

Reply:
To be honest, we could delete this code path completely since queue batch append isn't currently allowed

Reply:
I think it's worth keeping around for posterity sake until we split the CTC and can delete it completely. We don't want to forget that in our current design this codepath is absolutely required for the eventual proper functioning of the system.